cfgov/v1/models/filterable_list_mixins.py

```diff
@@ -1,17 +1,18 @@
-from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.core.paginator import Paginator
+from django.db import models
 
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
 from wagtail.contrib.routable_page.models import route
 from wagtail.models import Site
 from wagtailsharing.models import ShareableRoutablePageMixin
 
-from v1.atomic_elements.organisms import FilterableList
 from v1.documents import FilterablePagesDocumentSearch
 from v1.feeds import FilterableFeed
 from v1.models.learn_page import AbstractFilterPage
 from v1.util.ref import get_category_children
 
 
-class FilterableListMixin(ShareableRoutablePageMixin):
+class FilterableListMixin(ShareableRoutablePageMixin, models.Model):
```

Comment:
Is it still a mixin if it's an abstract base class? Should we change the name of this class to reflect that it inherits from `Model`?

Reply:
That is a reasonable request but also one that will require lots of changes across lots of files :) I remember having a similar idea, but I wasn't sure what the best name would be. We currently have an `AbstractFilterPage` which is the base class for _pages that can be filtered by another page_, so would this one be `AbstractFilteringPage`?

Reply:
I decided to go with `AbstractFilterablePage` for this, since we have `BrowseFilterablePage` and `SublandingFilterablePage`.